src/components/Inputs/Label/Label.js

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { Label as UIFabLabel } from 'office-ui-fabric-react/lib/Label';
+
+/** Label example */
+const label ="I am a label";
+function Label({required, disabled }) {
```

Comment:
Being a label, you should probably accept a 'label' prop to the component? Current example will always display "I am a label".

Reply:
Yeah